### DISCUSSION TITLE
Suchergebnisse aus Karte löschen

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#572] [Q&A] [answered]
Ich würde gerne die Unterkünfte aus der Karte löschen ohne die Route selbst zu löschen.
Wie bekomme ich die Suchergebnisse aus der Kartenansicht.
![Screenshot_20240616_215911_Cruiser](https://github.com/devemux86/cruiser/assets/124886589/02ebef93-edf1-4055-9fad-f81fb874f580)

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

Long press on one of the results and select "Delete all".

Drücken Sie lange auf eines der Ergebnisse und wählen Sie "Alle löschen".

**Reply by 1maxi2** [reply]
O.k. Danke funktioniert. Vergesse leider immer wieder das sich bei langen drücken oft eine Funktion verbirgt
